Wikipedia/Code/PreviewAndSaveViewController.h

```diff
@@ -1,4 +1,4 @@
-#import "Wikipedia-Swift.h"
```

Comment:
"Wikipedia-Swift.h" [shouldn't be put in header files of the same module to prevent cyclical dependencies](https://developer.apple.com/library/content/documentation/Swift/Conceptual/BuildingCocoaApps/MixandMatch.html#//apple_ref/doc/uid/TP40014216-CH10-ID136).